torch/_torch_docs.py

```diff
@@ -4220,6 +4220,28 @@
 
 """)
 
+add_docstr(torch.randn_like,
+           r"""
+randn_like(input, dtype=None, layout=None, device=None, requires_grad=False) -> Tensor
+
+Returns a tensor with the same size as :attr:`input` that is filled with
+random numbers from a normal distribution with zero mean and variance of
```

Comment:
is there a reason you write "zero" and "one" instead of 0 and 1?  That seems harder to read.

Reply:
Not in particular, no, will change.